backend/hitas/views/apartment_max_price.py

```diff
@@ -123,6 +128,28 @@ def update(self, request, *args, **kwargs):
             ampc.delete()
             return Response(status=HTTPStatus.NO_CONTENT)
 
+    @action(detail=True, methods=["GET"])
+    def download(self, request, **kwargs) -> HttpResponse:
+        with model_or_404(ApartmentMaximumPriceCalculation):
+            mpc = ApartmentMaximumPriceCalculation.objects.get(uuid=kwargs["pk"])
```

Comment:
Do we want to download a specific calculation as "confirmed max price calculation" or do we want to have a endpoint that returns the latest confirmed max price calculation? The difference being the calculation id does not have to be provided.

Reply:
Lets go with this for now and change it up if it's needed, but thats a nice idea 😄